<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrm</class>
 <widget class="QMainWindow" name="MainFrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Texture Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="mRenderViewFrame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="mImportTextureAction"/>
    <addaction name="separator"/>
    <addaction name="mExitApplicationAction"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="mStatusBar"/>
  <action name="mImportTextureAction">
   <property name="icon">
    <iconset resource="TextureTool.qrc">
     <normaloff>:/TextureToolRendering/ImportTextureIcon</normaloff>:/TextureToolRendering/ImportTextureIcon</iconset>
   </property>
   <property name="text">
    <string>Import textures</string>
   </property>
   <property name="statusTip">
    <string>Import a texture, or multiple</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="mExitApplicationAction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TextureTool.qrc"/>
 </resources>
 <connections/>
</ui>
